client/posts/render/image.ts

```diff
@@ -70,42 +70,30 @@ function renderFigcaption(post: Element, data: ImageData, reveal: boolean) {
         post.querySelector("header").after(el)
     }
 
-    const list: string[] = []
-    if (data.audio) {
-        list.push('\u266B')
-    }
-    if (data.length) {
-        list.push(readableLength(data.length))
-    }
-    if (data.apng) {
-        list.push('APNG')
-    }
-    list.push(readableFilesize(data.size), `${data.dims[0]}x${data.dims[1]}`)
-
     const [hToggle, , info, link] = Array.from(el.children) as HTMLElement[]
     if (!options.hideThumbs && !options.workModeToggle) {
         hToggle.hidden = true
     } else {
         hToggle.hidden = false
         hToggle.textContent = lang.posts[reveal ? 'hide' : 'show']
     }
-    info.textContent = `(${commaList(list)})`
+    info.querySelector(".media-title").textContent = data.title
+    info.querySelector(".media-artist").textContent = data.artist
+    info.querySelector(".has-audio").hidden = !data.audio
+    if (data.length) {
+        info.querySelector(".media-length").textContent = readableLength(data.length)
+    }
+    info.querySelector(".is-apng").hidden = !data.apng
+    info.querySelector(".filesize").textContent = readableFilesize(data.size)
+    info.querySelector(".dims").textContent = `${data.dims[0]}x${data.dims[1]}`
+    
```

Comment:
In hidden thumbnail mode. I guess rendering the whole thing is not strictly necessary, but there are too many things, that rerender an image and/or figcaption, and several flag combinations for each. To simplify this and avoid a lot of extra bookkeeping, I rerender the figure and figcaption on each change. Can be optimised away, of course.